packages/components/src/query-controls/index.native.js

```diff
@@ -79,7 +79,6 @@ const QueryControls = memo(
 						noOptionLabel={ __( 'All' ) }
 						selectedCategoryId={ selectedCategoryId }
 						onChange={ onCategoryChange }
-						hideCancelButton={ true }
```

Comment:
This doesn't exist.

It's passed all the way down to [<Select>](https://github.com/WordPress/gutenberg/blob/921135f9ec7f1e54030672eb7d11ca28034bf15b/packages/components/src/select-control/index.tsx#L110), which doesn't have a prop of `hideCancelButton`:

<img width="1190" alt="Screen Shot 2022-12-21 at 5 41 03 PM" src="https://user-images.githubusercontent.com/4063887/209024028-b5c947c3-6a2b-4b8f-b7a1-b5c18ce1bc28.png">

Reply:
Ah, good catch.

Added back in with https://github.com/WordPress/gutenberg/pull/46721/commits/7290d54471aed80abe3ba9c66ae7c00593c9396d.